### DISCUSSION TITLE
Compress "Timing Distribution" and "Accuracy Heatmap" on score details

### DISCUSSION THREAD

**Discussion by xMcSqueezy** [ppy/osu#27435] [Ideas]
Making those 2 sections in the same line would allow to fit all details on the same page, without needing to scroll down to see the rank/score/accuracy changes. I made a quick graphic showcasing the idea. I know it's scuffed but it gets my point accross.

![image](https://github.com/ppy/osu/assets/102478229/136acb5f-dc19-4bb0-a2e5-adb4565243dc)

**Comment by Joehuu** [comment]
There's https://github.com/ppy/osu/pull/24153 with the same positioning but it's old and has conflicts that it may need to be redone.

**Reply by xMcSqueezy** [reply]
Hopefully this post brings new attention to the idea 🙏
